backend/src/plugins/calibration/calibration_service_impl.h

```diff
@@ -169,6 +169,14 @@ class CalibrationServiceImpl final : public rpc::calibration::CalibrationService
         return grpc::Status::OK;
     }
 
+    grpc::Status Cancel(grpc::ServerContext * /* context */,
+                                const rpc::calibration::CancelCalibrationRequest * /* request */,
```

Comment:
I believe the indentation is wrong here. You probably want to run `make fix_style` to fix the indentation, otherwise the CI will fail :sweat_smile:.